src/controllers/update/update.review.beneficial.owner.other.ts

```diff
@@ -77,7 +77,6 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
 
     return res.redirect(UPDATE_BENEFICIAL_OWNER_TYPE_URL);
   } catch (error) {
-    console.log(error);
```

Comment:
😱

Reply:
no definitely keep, shouldn't have console.logs in there, my reaction was to its mere presence in the first place 😆